packages/components/pie-notification/src/defs-react.ts

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export type ReactBaseType = React.HTMLAttributes<HTMLDivElement>
```

Comment:
As the component is WIP, this definition should be revisited later

Reply:
Might be worth a to-do comment and a note in the notification epic? (Seems too small for its own ticket)

Reply:
Added a `TODO` comment